salt/orchestrate/mongodb/init.sls

```diff
@@ -0,0 +1,89 @@
+{% set subnet_ids = [] %}
+{% for subnet in salt.boto_vpc.describe_subnets(subnet_names=[
+    'public1-dogwood_qa', 'public2-dogwood_qa', 'public3-dogwood_qa']) %}
+{% do subnet_ids.append(subnet['id']) %}
```

Comment:
I ran into an issue with `u''` string prefixes until I updated this line to look like:
`{% do subnet_ids.append('{0}'.format(subnet['id'])) %}`